frontend/src/devicelist/SingleTableRow.js

```diff
@@ -1,61 +1,75 @@
-import React from 'react';
+import React from "react";
 import {
-    Box,
-    Collapse,
-    TableRow,
-    TableCell,
-    Typography,
-    IconButton,
-} from "@material-ui/core"
+  Box,
+  Collapse,
+  TableRow,
+  TableCell,
+  Typography,
+  IconButton,
+} from "@material-ui/core";
```

Comment:
Can we get ES Lint to be consistent with what it wants for the import formatting? Sometimes it wants all on one line, sometimes it's fine with stacked. Also I'm not a fan of comma on the last line, but that's me
![image](https://user-images.githubusercontent.com/35614138/99460278-92395580-28fd-11eb-941f-51e5bc81c665.png)

Reply:
The formatting is pretty much set to do this unless we allow for really long lines. I could not find an option to ignore max-length only for imports, so, I think it would be best to stick with the default here.